clients/web/src/models/CollectionCreationPolicyModel.js

```diff
@@ -0,0 +1,21 @@
+import _ from 'underscore';
+import AccessControlledModel from 'girder/models/AccessControlledModel';
+import { restRequest } from 'girder/rest';
+
+var CollectionCreationPolicyModel = AccessControlledModel.extend({
+    resourceName: 'system/setting/collection_creation_policy',
+    fetchAccess: function () {
+        return restRequest({
+            path: this.resourceName + '/access',
+            type: 'GET'
+        }).done(_.bind(function (resp) {
```

Comment:
Since we have ES6 features, the use of `_.bind` is (usually) not required, instead we should just use arrow functions.